english.tr

```diff
@@ -11,7 +11,7 @@ faq
 FAQ
 -----
 built_in_v
-Built in V
+Built-in V
```

Comment:
Built-in is like an adjective. "Built-in features" etc. This is a section for software built in V. So current version is correct.